docs/tasks/federation/set-up-placement-policies-federation.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Set up placement policies in Federation
+redirect_from:
+- "/docs/tutorials/federation/set-up-placement-policies-federation/"
+- "/docs/tutorials/federation/set-up-placement-policies-federation.html"
+---
+
+{% capture overview %}
+
+This page shows you can enforce policy-based placement decisions over Federated
+resources using an external policy engine.
+
+{% endcapture %}
+
+
+{% capture objectives %}
```

Comment:
Since this is a Task, you can remove the *objectives* section -- these are just restating the ToC anyway. Be sure to use the Task template at the end of the doc instead of the Tutorial one, i.e.

`{% include templates/task.md %}`